beast/physicsmodel/model_grid.py

```diff
@@ -433,20 +433,28 @@ def make_extinguished_sed_grid(
     g: SpectralGrid object
         spectral grid to transform
     """
+
+    # create the dust grid arrays so they can go into the beast_info file later
```

Comment:
Putting these into the beast_info file is not the only reason to create these arrays, but this is implied by this comment.  Maybe not needed to say that they are created for beast_info at all?  Implied as they are written to that file later?

Reply:
I fixed this as it was straightforward and this PR is needed so that others can avoid the error fixed by this PR.